fedimint-ui/README.md

```diff
@@ -7,12 +7,12 @@ This project includes the following apps / packages:
 ### Apps
 
 - `guardian-ui`: Web app experience for setting up and administering fedimints. This is used by the Fedimint guardians
-- `gateway-ui`: Web app experience for managing Fedimint gateways
+- `gateway-ui`: Web app experience for managing Fedimint gateways. This is used by Gateway administrators
 
 ### Packages
 
 - `ui`: Shared React UI component library for building Fedimint UI experiences
-- `eslint-config-custom`: Shared `eslint` configurations (includes `eslint-plugin-react` and `eslint-config-prettier`)
+- `eslint-config`: Shared `eslint` configurations (includes `eslint-plugin-react` and `eslint-config-prettier`)
 - `tsconfig`: Shared `tsconfig.json`s used throughout Fedimint UI apps
 
 ## Development
```

Comment:
```suggestion
## Development

From root repo directory:

1. `cd fedimint-ui`
1. `nix develop .#fedimint-ui`
1. `yarn install` (First time only)
1. You can run any of the following commands from `fedimint-ui/` directory
```

Because although it's a given, we all forget the first time. 😅

Reply:
ack